mpc:

  horizon: 5

  enable_learning_process: true

  feedback_gain:
    data: [1.0000, 1.0000]
    
  nominal_feedback_gain:
    data: [1.0000, 1.0000]

  optimizer:
    states_error_weight_matrix:
      data: [1.0000, 0.0000, 0.0000, 1.0000]

    input_error_weight_matrix:
      data: [1.0000]

    terminal_state_weight_matrix:
      data: [2.0000, 0.0000, 0.0000, 2.0000]
